### PR TITLE
FIX Reinstate tab validation icons

### PR DESCRIPTION
**Pull request by emteknetnz**
Tab validation icons were accidentally removed in https://github.com/silverstripe/silverstripe-admin/pull/1164

This reinstates them, as well as adding behat tests which is for https://github.com/silverstripe/silverstripe-admin/issues/1122

### CODE REVIEW

File: tests/behat/src/AdminContext.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            assertNotNull($element, $message);
+            assertTrue(strpos($element->getText(), $text) !== false, $message);
+        }
+    }
+
+    /**
+     * @When /^I click the "([^"]+)" element$/
+     * @param $selector
+     */
+    public function iClickTheElement($selector)
```
Comment (Cheddam) [inline tests/behat/src/AdminContext.php:71]: I'm surprised this doesn't already exist? Surely other behat tests have clicked non-button elements before?

File: tests/behat/src/AdminContext.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SilverStripe\Admin\Tests\Behat\Context;
+
+use Behat\Behat\Context\Context;
+use Behat\Mink\Element\DocumentElement;
+use SilverStripe\BehatExtension\Context\MainContextAwareTrait;
+
+/**
+ * Context used to create fixtures in the SilverStripe ORM.
```
Comment (Cheddam) [inline tests/behat/src/AdminContext.php:10]: Is this accurate? 🤔

File: tests/behat/src/AdminContext.php (hunk 1 above)
Comment (emteknetnz) [reply tests/behat/src/AdminContext.php:71]: It does exist in 2 or 3 other contexts, however it's not at a "high level" e.g. in behat-extension.  It's less than ideal though pragmatically it really doesn't matter

File: tests/behat/src/AdminContext.php (hunk 2 above)
Comment (emteknetnz) [reply tests/behat/src/AdminContext.php:10]: no :p

Comment (emteknetnz) [reply tests/behat/src/AdminContext.php]: Have removed

**Review by Cheddam — APPROVED** [review]
MOG

**Pull request merged**